app/src/main/kotlin/com/hippo/ehviewer/ui/reader/PagerItem.kt

```diff
@@ -82,18 +85,29 @@ fun PagerItem(
                     }
                 }
             }
-            Image(
-                painter = painter,
-                contentDescription = null,
-                modifier = contentModifier.fillMaxSize(),
-                contentScale = contentScale,
-                colorFilter = when {
-                    grayScale && invert -> grayScaleAndInvertFilter
-                    grayScale -> grayScaleFilter
-                    invert -> invertFilter
-                    else -> null
-                },
-            )
+            if (image.hasQRCode) {
+                val bgColor by collectBackgroundColorAsState()
+                AsyncImage(
+                    model = ADSPlaceholderFile,
+                    error = ColorPainter(bgColor),
+                    contentDescription = null,
+                    modifier = contentModifier.fillMaxSize(),
+                    contentScale = contentScale,
+                )
```

Comment:
A text to indicate that the image is blocked if there is no custom placeholder.

Reply:
Done